.github/workflows/release.yml

```diff
@@ -36,9 +36,10 @@ jobs:
 
       - name: Prepare repository
         run: git fetch --unshallow --tags
-
+      - run: corepack enable
+      - run: yarn set version berry
       - name: Install dependencies
-        run: yarn install --ignore-scripts
```

Comment:
Any particular reason for dropping this? Did you notice a behavior change?

Reply:
Jarel and I talked and yarn berry does not support that flag `--ignore-scripts` and we don't think we need it.